components/support/nimbus-fml/src/parser.rs

```diff
@@ -36,30 +39,46 @@ pub(crate) struct FieldBody {
 }
 
 #[derive(Debug, Deserialize, Serialize, Clone)]
+#[serde(deny_unknown_fields)]
 pub(crate) struct ObjectBody {
     description: String,
     failable: Option<bool>,
     fields: HashMap<String, FieldBody>,
 }
 
 #[derive(Debug, Deserialize, Serialize, Clone, Default)]
+#[serde(deny_unknown_fields)]
```

Comment:
Looking at https://serde.rs/field-attrs.html#flatten, it's saying flatten shouldn't be used with `deny_unknown_fields`, not sure what the implication is but we seem to have good test coverage so 🤷